packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -214,6 +214,7 @@ export const Webchat = forwardRef((props, ref) => {
     setCurrentAttachment,
     // eslint-disable-next-line react-hooks/rules-of-hooks
   } = props.webchatHooks || useWebchat()
+  const firstUpdate = useRef(true)
```

Comment:
Why are you using `useRef`? Wouldn't `useState` be more appropiate in this case?

Reply:
here the suggested solution: https://medium.com/anna-coding/the-way-to-check-if-its-the-first-time-for-useeffect-function-is-being-run-in-react-hooks-170520554067